pegasus/helpers/section_api_helpers.rb

```diff
@@ -21,20 +21,19 @@ def self.fetch_user_students(user_id)
   end
 
   def self.create(params)
-    name = params[:name].to_s
-    name = 'New Student' if name.empty?
-
-    params[:gender] = nil unless valid_gender?(params[:gender])
-    params[:birthday] = age_to_birthday(params[:age]) if age_to_birthday(params[:age])
+    name = params[:name].to_s.present? ? params[:name].to_s : 'New Student'
```

Comment:
That behaves differently as (for example) `nil.to_s` yields `''` (the empty string) and `'' || 'New Student' yields the empty string. Good thought, though!